setup_payram.sh

```
@@ -775,6 +775,47 @@ check_disk_space_requirements_silent() {
+  local check_cmd=""
+  if command -v ss >/dev/null 2>&1; then
+    check_cmd="ss -tuln"
+  elif command -v netstat >/dev/null 2>&1; then
+    check_cmd="netstat -tuln"
+  else
+    log "WARN" "Neither 'ss' nor 'netstat' available - skipping port check"
+    return 0
+  fi
+  
+  for port in "${ports[@]}"; do
+    if $check_cmd 2>/dev/null | grep -E ":$port[[:space:]]|:$port$" >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)